docs/docs/getting-started/6-nocode-docker-deployment.md

```diff
@@ -0,0 +1,38 @@
+# No-Code Deployment with Docker
```

Comment:
I see the other repo describes itself as "Detector Builder" now, which I think is a better name.  When I think about the most important ideas to convey about this option, "Docker" isn't high on the list.  If I was going to include more details, I'd say that it's a web app.  Or maybe just "No-code detector builder".